ios/FluentUI/List/FluentList.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// This a wrapper around `SwiftUI.List` that has fluent style applied. It is intended to be used in conjunction with `FluentUI.FluentListSection` and `FluentUI.ListItem`
+/// to provide a completely fluentized list, however, it can be used on it's own if desired.
+///
+/// This component is a work in progress. Expect changes to be made to it on a somewhat regular basis.
+public struct FluentList<ListContent: View>: View {
+
+    // MARK: Initializer
+
+    /// Creates a `FluentList`
+    /// - Parameters:
+    ///   - content: SwiftUI content to show inside of the list.
+    public init(@ViewBuilder content: @escaping () -> ListContent) {
+        self.content = content
+    }
+
+    public var body: some View {
+        List {
+            content()
+        }
+        .listStyle(.insetGrouped)
```

Comment:
similar to tvc and list item, do we also want to support plain, grouped style?

Reply:
Good idea. Updated to take an optional list style and updated the demo controllers. See demo video in validation section.

Reply:
I think this would be better as a modifier? So we have some default value, but then could set the value in a modifier. Makes it not an init and just add if we dont want the default style. (Like what is done in `ListItemModifiers.swift` as one example)

Reply:
wait I just realized. this is just a system modifier so we don’t even need to set this right? leave it up to the client to set the style. unless we wanted to enforce fluent background colors

Reply:
yeah we will be enforcing a fluent color for the background

Reply:
Okay, more good ideas. I realized the awkwardness and misalignment with Apple of the style param in the init shortly after posting the revision. I've taken Alex's suggestion and moved the list style to a view modifier for `FluentList`. It and the custom style enum should make supporting the fluent background color easier.